bitcoind-wallet/bin/cli.js

```diff
@@ -11,7 +11,8 @@ const cli = meow(`
 
   Commands
     sendToAddress         [address] [btc] Send btc to an address
-    getNewAddress         Get an address to receive btc
+    getNewAddress         [addressType=bech32] Get an address to receive btc (optional)
```

Comment:
I find this `(optional)` indicator awkwardly placed. It seems ambiguous whether or not the address type is optional, or receiving btc is optional (as funny as that is).

I had a quick read through https://en.wikipedia.org/wiki/Usage_message and https://developers.google.com/style/code-syntax and it seems like the convention is to use square brackets to mark optionality.

That would mean `[addressType=bech32]` is well-formatted but potentially confusing, as some users might try to type

`getNewAddress addressType=bech32` instead of `getNewAddress bech32`.

With all that in mind, I think that we should make the following changes:

```
    sendToAddress         <address> <btc> Send btc to an address
    getNewAddress         [address_type] Get an address to receive btc
                            Values: legacy, p2sh-segwit, bech32 (default)
    sendRawTransaction    <transaction> Broadcast a raw transaction
    getBalance            <address> Get the balance of an address
```

Reply:
At the same time, I think this PR is an [improvement](https://google.github.io/eng-practices/review/reviewer/standard.html), so I'll merge as is, and we can decide to fiddle with the help syntax separately.

Reply:
Thanks for the research! I was struggling to find some directions on how to format the help message properly here. I was also thinking about using [yargs](https://github.com/yargs/yargs) for the cli.